src/bundle/Resources/views/dashboard/tab/all_media.html.twig

```diff
@@ -17,8 +17,12 @@
                 <td><a href="{{ url('_ez_content_view', { 'contentId': row.contentId }) }}">{{ row.name }}</a></td>
                 <td>{{ row.type }}</td>
                 <td>
-                    {{ ez_render_field(row.contributor, 'first_name') }}
-                    {{ ez_render_field(row.contributor, 'last_name') }}
+                    {% if row.contributor is not null %}
+                        {{ ez_render_field(row.contributor, 'first_name') }}
```

Comment:
I actually don't like the idea of hardcoding `first_name` and `last_name` as as for now any content type can represent an user as long as a it has `ezuser` fieldtype. You are meant to use `getName()` method to get a proper name of the user. You can reference User Menu: https://github.com/ezsystems/ezplatform-admin-ui/blob/df1e4e7704b2d4749f1a7f4bed0a3d5f7205bf98/src/bundle/Resources/views/parts/navigation.html.twig#L21